cmd/uplink/cmd/cp.go

```diff
@@ -84,20 +82,30 @@ func upload(ctx context.Context, src fpath.FPath, dst fpath.FPath, showProgress
 		return fmt.Errorf("source cannot be a directory: %s", src)
 	}
 
-	metainfo, streams, err := cfg.Metainfo(ctx)
+	project, err := cfg.GetProject(ctx)
 	if err != nil {
 		return err
 	}
+	defer func() {
+		err = project.Close()
+		if err != nil {
+			fmt.Printf("Error closing project uplink: %+v\n", err)
+		}
+	}()
 
-	createInfo := storj.CreateObject{
-		RedundancyScheme: cfg.GetRedundancyScheme(),
-		EncryptionScheme: cfg.GetEncryptionScheme(),
-		Expires:          expiration.UTC(),
-	}
-	obj, err := metainfo.CreateObject(ctx, dst.Bucket(), dst.Path(), &createInfo)
+	var access libuplink.EncryptionAccess
+	copy(access.Key[:], []byte(cfg.Enc.Key))
+
+	bucket, err := project.OpenBucket(ctx, dst.Bucket(), &access, 0)
```

Comment:
The last argument "0" was removed. Merge the latest master to get the latest updates in libuplink.